src/shims/recompose/index.js

```diff
@@ -0,0 +1,11 @@
+/** This file is an auto-generated shim, aliased in for "recompose" in the webpack config.
+*  When components import 'recompose', we want to give them back the copy
+*  Zimbra passed down when it called the factory provided to zimlet().
+*/
+
+/* eslint-disable camelcase, dot-notation */
+import { warnOnMissingExport } from '../';
+const wrap = warnOnMissingExport.bind(null, global.shims['recompose'], 'recompose');
+
+export default wrap('default');
```

Comment:
ideally this should list down all recompose methods

Reply:
I have purposely added this entire package to the shims. Wouldn't we want to expose entire recompose module to zimlets? If not then we will have to modify zm-x-web to add a shim everytime a zimlet developer needs some utility from recompose. Kindly advise.

Reply:
I have removed 'recompose' entry from build shims. Added recompose as dev-dependency in package.json so that when build runs it will pick up all methods from recompose in shimmed modules.